### PR TITLE
content/learn.md add 2 new QA

### PR DESCRIPTION
**Pull request by MateuszKochner**
Signed-off-by: Mateusz Kochner <mateusz.kochner@3mdeb.com>

### CODE REVIEW

File: content/learn.md [hunk 1]
```diff
@@ -247,6 +247,10 @@ dasharo_faq:
 
         If you don’t have a solution yet, let Dasharo become your answer. [Contact us](/#get-a-quote)
 
+    - title: What kind of consulting services that 3mdeb can provide other than online course?
+      content: >-
+        In addition to training, we provide in-house development and maintenance to keep hardwar and softwar up to date, or provide ad-hoc support to your development team.
```
Comment (pietrushnic) [inline content/learn.md:252]: content is generally not wrapped in this file, but can we start wrapping new content?

**Comment by pietrushnic** [comment]
@artur-rs we probably need pre-commit for this respository.

@MateuszKochner your content contain simple grammar and typo mistakes. Please use spellchecker before pushing your changed for review. Thank you.

File: content/learn.md (hunk 1 above)
Comment (MateuszKochner) [reply content/learn.md:252]: @pietrushnic Automatic wrapping (Alt+Q) dosen't wok, We would have to do it manually... worth it?

Comment (pietrushnic) [reply content/learn.md:252]: @MateuszKochner I have no idea what editor you use. My editor don't have any feature under Alt+Q.  Tools should not  prevent high quality content preparation. Please correctly configure tools you using.

I'm repeated argument about 80 character wrapping many times: https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings

This request is not about whim, but the way most programmers in open-source software world trained their brains.  Reviewing long lines is harder and slower.

Comment (artur-rs) [reply content/learn.md]: @pietrushnic after wrapping all sections, render was broken (the point is we are using markdown header styling, not markdown content). I don't think effort of wrapping everything in this document is needed here.

File: content/learn.md [hunk 2]
```diff
@@ -247,6 +247,28 @@ dasharo_faq:
...
+      content: >-
+         In addition to training, we provide in-house development and maintenance to keep hardware and software up to date or provide ad-hoc support to your development team.
+
+    - title: Where I can find your GitHub/issues?
+      content: >-
+        - Visit [github.com/Dasharo](https://github.com/Dasharo) where you can find details about our company and repositories.
+
+        - [Matrix Dasharo Space] is a place where you can find solutions to many problems and if not - our community will gladly help you. We encourage you to troubleshoot your open-source firmware issues there. Here is the invitation [Link](https://matrix.to/#/#dasharo:matrix.org).
+
+    - title: Can I have got Dasharo individually?
```
Comment (artur-rs) [inline content/learn.md:260]:
```suggestion
    - title: Can I get the Dasharo individually?
```

File: content/learn.md [hunk 3]
```diff
@@ -247,6 +247,28 @@ dasharo_faq:
...
+      content: >-
+        Yes, you can get it four ways:
+        
+        - Join the newsletter with the latest firmware realease
```
Comment (artur-rs) [inline content/learn.md:264]: @MateuszKochner we should point to where user can subscribe

File: content/learn.md [hunk 4]
```diff
@@ -247,6 +247,28 @@ dasharo_faq:
...
+
+        - [Matrix Dasharo Space] is a place where you can find solutions to many problems and if not - our community will gladly help you. We encourage you to troubleshoot your open-source firmware issues there. Here is the invitation [Link](https://matrix.to/#/#dasharo:matrix.org).
+
+    - title: Can I have got Dasharo individually?
+      content: >-
+        Yes, you can get it four ways:
+        
+        - Join the newsletter with the latest firmware realease 
+
+        - Find firmware links products on our and client's websites
```
Comment (artur-rs) [inline content/learn.md:266]: @MateuszKochner add link from the 3rd point here, as a `on our` method, then remove 3rd method. Use also more detailed description: try to help where users can find the download links (client's websites with desired products which supports Dasharo firmware). Example from NovaCustom should be nice.

File: content/learn.md [hunk 5]
```diff
@@ -247,6 +247,28 @@ dasharo_faq:
...
+
+        - Find "Supported hardware" on [Docs Dasharo](https://docs.dasharo.com/)
+
+        - Do it yourself, visit [github.com/Dasharo](https://github.com/Dasharo)
```
Comment (artur-rs) [inline content/learn.md:270]:
```suggestion
        - Build it yourself, visit [github.com/Dasharo](https://github.com/Dasharo)
```
Plus let's point to the specific repository with coreboot fork.

File: content/learn.md [hunk 6]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+      content: >-
+        First, there is no license fee. By choosing the support package, we start cooperation for a longer period of at least half a year. The plan is presented at the beginning of each month. The payment depends on the senior developer's time worked. exceeding the working time inconsistent with the assumed plan is reported to the client. We also assume time for management and meetings with the client, which is necessary to run the project.
+
+    - title: What pricing model do you offer?
+      content: >-
+        Dasharo’s pricing model does not rely on volume and since most components are open-source, you can always change vendor and re-own your firmware code. Dasharo can be purchased by OEM/ODM providers, community crowdfunding, individuals, companies that are looking for dedicated/confidential firmware solutions. To get more dedicated payment information, fill [Get a quote](/#get-a-quote).
+
+    - title: Can I introduce UEFI firmware into my existing platform/estate?
+      content: >-
+        i dont know
```
Comment (artur-rs) [inline content/learn.md:378]: @MateuszKochner leave `TBD` in such cases

File: content/learn.md [hunk 7]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+# odwołanie do pytania Can I have got Dasharo individually?
```
Comment (artur-rs) [inline content/learn.md:379]: @MateuszKochner do not use polish here

File: content/learn.md [hunk 8]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+    - title: Can I implement Coreboot (as boot fw) + iPXE (as payload) to my device?
+      content: >-
+        i dont know
```
Comment (artur-rs) [inline content/learn.md:382]: @MateuszKochner leave `TBD` in such cases

File: content/learn.md [hunk 9]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+# odwołanie do pytania Can I have got Dasharo individually?
```
Comment (artur-rs) [inline content/learn.md:383]: @MateuszKochner do not use polish here

File: content/learn.md [hunk 10]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+    - title: Can I port Core boot to use in my production environment?
+      content: >-
+# odwołanie do pytania Can I have got Dasharo individually?
```
Comment (artur-rs) [inline content/learn.md:386]: @MateuszKochner do not use polish here. Will you add such reference?

File: content/learn.md [hunk 11]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+
+    - title: What pricing model do you offer?
+      content: >-
+        Dasharo’s pricing model does not rely on volume and since most components are open-source, you can always change vendor and re-own your firmware code. Dasharo can be purchased by OEM/ODM providers, community crowdfunding, individuals, companies that are looking for dedicated/confidential firmware solutions. To get more dedicated payment information, fill [Get a quote](/#get-a-quote).
+
+    - title: Can I introduce UEFI firmware into my existing platform/estate?
+      content: >-
+        i dont know
+# odwołanie do pytania Can I have got Dasharo individually?
+    - title: Can I implement Coreboot (as boot fw) + iPXE (as payload) to my device?
```
Comment (artur-rs) [inline content/learn.md:380]:
```suggestion
    - title: Can I implement coreboot (as boot fw) + iPXE (as payload) to my device?
```

File: content/learn.md [hunk 12]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+        i dont know
+# odwołanie do pytania Can I have got Dasharo individually?
+    - title: Can I port Core boot to use in my production environment?
+      content: >-
+# odwołanie do pytania Can I have got Dasharo individually?
+    - title: Can I design a BIOS base system or pre-boot software or environment that would forcefully install our management software itself
+      content: >-
+        Yes, this is possible.
+
+    - title: Can I develop, maintain and support such a software for common laptops, desktops produced by OEMs such as HP, Dell, and Lenovo?
```
Comment (artur-rs) [inline content/learn.md:391]:
```suggestion
    - title: Can 3mdeb develop, maintain and support software for common laptops or desktops produced by OEMs such as HP, Dell, and Lenovo?
```

File: content/learn.md [hunk 13]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+      content: >-
+        Yes, we would be but details of the such project should be discussed. How we
```
Comment (artur-rs) [inline content/learn.md:393]:
```suggestion
        Yes, but details of such project should be discussed beforehand. How we
```

File: content/learn.md [hunk 14]
```diff
@@ -341,8 +363,48 @@ dasharo_faq:
...
+        typically operate:
```
Comment (artur-rs) [inline content/learn.md:394]: @MateuszKochner maybe it's better to link our sales pipeline from 3mdeb website?

File: content/learn.md [hunk 15]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
-      
+        Typically, firmware solutions are sold through royalty payment or subscription-based models. The first option relies on the volume of offered products, what effectively eliminates industries with low volume but high-value platforms e.g. medical robots. Subscription-based models, although better, are still tied to firmware solution providers and similarly lead to all the scope of negative effects of vendor lock-in. 
+
+    - title: What services include in the price plan?
```
Comment (artur-rs) [inline content/learn.md:398]:
```suggestion
    - title: What services are included in the price plan?
```

File: content/learn.md [hunk 16]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+      content: >-
+        First, there is no license fee. By choosing the support package, we start cooperation for a longer period of at least half a year. The plan is presented at the beginning of each month. The payment depends on the senior developer's time worked. exceeding the working time inconsistent with the assumed plan is reported to the client. We also assume time for management and meetings with the client, which is necessary to run the project.
```
Comment (artur-rs) [inline content/learn.md:400]:
```suggestion
        First, there is no license fee. By choosing the support package, we start cooperation for a longer period of at least half a year. The plan is presented at the beginning of each month. The payment depends on the senior developer's time worked. The amount of remaining budget is reported at least once a month via e-mail message to the Client. We also assume time for management and status/planning meetings with the client.
```

File: content/learn.md [hunk 17]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+
+    - title: What pricing model do you offer?
+      content: >-
+        Dasharo’s pricing model does not rely on volume and since most components are open-source, you can always change vendor and re-own your firmware code. Dasharo can be purchased by OEM/ODM providers, community crowdfunding, individuals, companies that are looking for dedicated/confidential firmware solutions. To get more dedicated payment information, fill [Get a quote](/#get-a-quote).
+
+    - title: Can I introduce UEFI firmware into my existing platform/estate?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
```
Comment (artur-rs) [inline content/learn.md:408]: @MateuszKochner that's not a correct link - see the examples above, like in the `Get a quote`.

File: content/learn.md [hunk 18]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+
+    - title: Can I implement coreboot (as boot fw) + iPXE (as payload) to my device?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
```
Comment (artur-rs) [inline content/learn.md:412]: @MateuszKochner that's not a correct link - see the examples above, like in the `Get a quote`.

File: content/learn.md [hunk 19]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+
+    - title: Can I port Core boot to use in my production environment?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
```
Comment (artur-rs) [inline content/learn.md:416]: @MateuszKochner that's not a correct link - see the examples above, like in the `Get a quote`.

File: content/learn.md [hunk 20]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+
+    - title: Can I design a BIOS base system or pre-boot software or environment that would forcefully install our management software itself
```
Comment (artur-rs) [inline content/learn.md:418]:
```suggestion
    - title: Can I design a BIOS base system or pre-boot software or environment that would forcefully install our management software itself?
```

File: content/learn.md [hunk 21]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+      content: >-
+        Yes, this is possible.
```
Comment (artur-rs) [inline content/learn.md:420]:
```suggestion
        Yes, this is possible. Technical details should be discussed on the dedicated meeting.
```

Comment (MateuszKochner) [reply content/learn.md:420]: poprawiłem

File: content/learn.md (hunk 20 above)
Comment (MateuszKochner) [reply content/learn.md:418]: poprawiłem

File: content/learn.md (hunk 18 above)
Comment (MateuszKochner) [reply content/learn.md:412]: corrected

File: content/learn.md (hunk 19 above)
Comment (MateuszKochner) [reply content/learn.md:416]: corrected

File: content/learn.md (hunk 17 above)
Comment (MateuszKochner) [reply content/learn.md:408]: corrected

File: content/learn.md (hunk 15 above)
Comment (MateuszKochner) [reply content/learn.md:398]: corrected

File: content/learn.md [hunk 22]
```diff
@@ -247,6 +247,58 @@ dasharo_faq:
...
+
+        - [NovaCustom NV4x 11th Generation Dasharo](https://newsletter.3mdeb.com/subscription/S5ze5u_qN)
+
+        - [NovaCustom NV4x 12th Generation Dasharo](https://newsletter.3mdeb.com/subscription/ZkbNv4qdO)
+
+        - [PC Engines Open Source Firmware Release](https://newsletter.3mdeb.com/subscription/ReBpt3IZY)
+
+         Build it yourself, visit
+
+        - [github.com/Dasharo](https://github.com/Dasharo)
```
[the remark this answers is not in the document]
Comment (MateuszKochner) [reply content/learn.md:300]: corrected

File: content/learn.md [hunk 23]
```diff
@@ -247,6 +247,58 @@ dasharo_faq:
 
         If you don’t have a solution yet, let Dasharo become your answer. [Contact us](/#get-a-quote)
 
+    - title: What consulting services can 3mdeb provide beyond online courses?
+      content: >-
+         In addition to training, we provide in-house development and maintenance to keep hardware and software up to date or provide ad-hoc support to your development team.
+
+    - title: Where I can find your GitHub/issues?
+      content: >-
+        - Visit [github.com/Dasharo](https://github.com/Dasharo) where you can find details about our company and repositories.
```
[the remark this answers is not in the document]
Comment (MateuszKochner) [reply content/learn.md:256]: added

File: content/learn.md [hunk 24]
```diff
@@ -341,8 +393,41 @@ dasharo_faq:
...
+
+    - title: Can I introduce UEFI firmware into my existing platform/estate?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
+
+    - title: Can I implement coreboot (as boot fw) + iPXE (as payload) to my device?
+      content: >-
+        [Find out here](//localhost:1313/pages/learn/#q-general-9)
+
+    - title: Can I port Core boot to use in my production environment?
```
[the remark this answers is not in the document]
Comment (MateuszKochner) [reply content/learn.md:414]: corrected

File: content/learn.md (hunk 1 above)
Comment (pietrushnic) [reply content/learn.md]: @artur-rs it is very hard to me to do review in code that has not wrapped lines. I will not insist if this breaks wesbite, but it doesn't mean I think it is ok. Disagree and commit.

**Comment by artur-rs** [comment]
New bug found: https://github.com/Dasharo/dasharo-website/issues/36

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**